cnd/src/http_api/route_factory.rs

```diff
@@ -133,6 +133,14 @@ pub fn create(
         .and(facade.clone())
         .and_then(http_api::routes::index::post_halight_bitcoin_herc20);
 
+    // This will replace get_halight_swap once implemented.
```

Comment:
🔧 Seems obvious, could just remove these comments?

Reply:
Oh, you thought that was obvious?  I thought `get_halight_swap` was so mis-appropriately named that it was not obvious at all :)